std_msgs/Header header

float64 gps_time_week
float64 latitude
float64 longitude
float64 altitude
float64[3] velocity_ned
float64[4] quaternion
float64[3] acceleration
float64[3] rotation_rate
float64[3] position_rms
float64[3] velocity_rms
float64[3] attitude_rms
uint16 gps_week_number
uint8 alignment


